docs/components/index.md

```diff
@@ -116,38 +116,79 @@ Data binding works with [DateTime](https://docs.microsoft.com/dotnet/api/system.
 }
 ```
 
+The `bind` Tag Helper uses the `format-value` attribute to apply the date format to the `value` node of the `input` element. The format is also applied by the value handler of the `onchange` node.
```

Comment:
The `format-value` attribute specifies the date format to apply to the `value` of the `input` element. The format is also used to parse the value when an `onchange` event occurs.